internal/appsec/waf.go

```diff
@@ -210,9 +227,23 @@ func newHTTPWAFEventListener(handle *waf.Handle, addresses []string, timeout tim
 			values := make(map[string]interface{}, len(addresses))
 			for _, addr := range addresses {
 				switch addr {
-				case serverRequestBodyAddr:
-					if body != nil {
-						values[serverRequestBodyAddr] = body
+				case serverRequestRawURIAddr:
```

Comment:
These shouldn't be added here. The only addresses left here should be the response related ones, for which we don't block.

Reply:
Rebase issue